ca/client_test.go

```diff
@@ -1118,9 +1125,10 @@ func TestClient_SSHBastion(t *testing.T) {
 					t.Errorf("Client.SSHBastion() = %v, want nil", got)
 				}
 				if tt.responseCode != 200 {
-					sc, ok := err.(render.StatusCodedError)
-					assert.Fatal(t, ok, "error does not implement StatusCodedError interface")
-					assert.Equals(t, sc.StatusCode(), tt.responseCode)
+					var sc render.StatusCodedError
+					if assert.True(t, errors.As(err, &sc)) {
```

Comment:
In some places the `assert.True()` was provided with the `"error does not implement StatusCodedError interface"` msg. For consistency I think it's good to have them in all.